README.md

```diff
@@ -258,6 +258,8 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 
 | OpenTelemetry Operator | Kubernetes           | Cert-Manager         |
 |------------------------|----------------------|----------------------|
+| v0.48.0                | v1.19 to v1.23       | 1.6.1                |
```

Comment:
There should not be changes in this support table.

Reply:
I rebased. I will let you add documentation on your convenience (I already added it then remove)